### ISSUE TITLE
UI Enhancement

### ISSUE THREAD

**Issue by debidattasuryaprakash** [Spectrum-CETB/2-Degree#1]
New contributors are requested to contribute to the following features:
- [ ] Navbar Modification
- [ ] Responsive UI Design
- [ ] Adding card carousel to the video section
- [ ] Adding new UI Cards (preferred Glassmorphism/ Neomorphism)

**Comment by cantidosan** [comment]
Hello, I'd love to be an assignee, the Responsive UI design would be great practice.

**Comment by debidattasuryaprakash** [comment]
@cantidosan Go For it and try to maintain proper file structure

**Comment by SugarZ3ro** [comment]
Can I work on it too?

**Comment by debidattasuryaprakash** [comment]
Both have been assigned @cantidosan and @Obito2002. Continue working👌

**Comment by debidattasuryaprakash** [comment]
Any Updates @cantidosan and @Obito2002

**Comment by cantidosan** [comment]
yeah,for sure, PR tomorrow during my stream. Nervous :) newb i am
